defender/tests.py

```diff
@@ -13,7 +13,7 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.models import AnonymousUser
 from django.contrib.sessions.backends.db import SessionStore
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```

Comment:
This seems to be failing when run against old versions of django. Maybe we can do the following.


```
try:
    django.urls import reverse
except ImportError:
    django.core.urlresolvers import reverse
```

It tries to do it the new way, if it fails it falls back to the old way.